mlflow/ml-package-versions.yml

```diff
@@ -317,3 +317,15 @@ spark:
     run: |
       # TODO: Add datasource autologging tests
       find tests/spark_autologging/ml -name 'test*.py' | xargs -L 1 pytest --large
+
+spark.datasource:
+  package_info:
+    pip_release: "pyspark"
+    install_dev: |
+      ./dev/setup-spark-datasource-autologging.sh
+
+  autologging:
+    requirements: ["scikit-learn"]
+    run: |
+      export SPARK_HOME=/tmp/spark-3.0.0-preview-bin-hadoop2.7
+      find tests/spark_autologging/datasource -name 'test*.py' | xargs -L 1 pytest --large
```

Comment:
```suggestion
    run: |
      # TODO: Add datasource autologging tests
      find tests/spark_autologging/ml -name 'test*.py' | xargs -L 1 pytest --large
      
      if [ "$pyspark_version" == "3.0.0" ]; then
        ./dev/setup-spark-datasource-autologging.sh
        export SPARK_HOME=/tmp/spark-3.0.0-preview-bin-hadoop2.7
        find tests/spark_autologging/datasource -name 'test*.py' | xargs -L 1 pytest --large
      fi
```

Reply:
@dbczumar This might be better than creating two spark workflows. What do you think?